test/modules/SessionReplay.spec.ts

```diff
@@ -29,4 +29,10 @@ describe('Session Replay Module', () => {
     expect(NativeSessionReplay.setUserStepsEnabled).toBeCalledTimes(1);
     expect(NativeSessionReplay.setUserStepsEnabled).toBeCalledWith(true);
   });
+
+  it('should call the native method getSessionReplayLink', () => {
+    SessionReplay.getSessionReplayLink();
+
+    expect(NativeSessionReplay.getSessionReplayLink).toBeCalledTimes(1);
```

Comment:
I think we also need to assert that it returns whatever value `NativeSessionReplay.getSessionReplayLink` returns which means we also need to mock the returned value of `NativeSessionReplay.getSessionReplayLink` then verify that `SessionReplay.getSessionReplayLink` returns the same value.

What do you think?